lib/update/cluster/steps.go

```diff
@@ -71,7 +71,15 @@ func (r *phaseBuilder) initSteps(ctx context.Context) error {
 	if err != nil {
 		return trace.Wrap(err)
 	}
-	etcd, err := shouldUpdateEtcd(installedRuntimeApp, r.updateRuntimeApp, r.packages)
+	installedRuntimePackage, err := schema.GetDefaultRuntimePackage(r.installedApp.Manifest)
```

Comment:
Does this properly handle custom planets? It's just the name "Get**Default**RuntimePackage" gives the impression that it always uses the default planet.

Reply:
Yeah, it is confusing. I think I'll drop the default part.